docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,196 @@
+---
+id: adjusting-bpmn-models
+title: Adjusting BPMN models
+description: "Learn how to adjust your BPMN models when migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+To migrate BPMN process models from Camunda Platform 7 to Camunda Platform 8, you must adjust them:
+
+- The namespace of extensions has changed (from `http://camunda.org/schema/1.0/bpmn` to `http://camunda.org/schema/zeebe/1.0`)
+- Different configuration attributes are used
+- Camunda Platform 8 has a _different coverage_ of BPMN elements (see [Camunda Platform 8 BPMN coverage](/components/modeler/bpmn/bpmn-coverage.md) vs [Camunda Platform 7 BPMN coverage](https://docs.camunda.org/manual/latest/reference/bpmn20/)), which might require some model changes. Note that the coverage of Camunda Platform 8 will increase over time.
+
+The following sections describe what the existing community supported [Backend diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter) does by BPMN symbol and explain unsupported attributes.
+
+### General considerations
+
+The following attributes/elements cannot be migrated:
+
+- `camunda:asyncBefore`: Every task in Zeebe is always asyncBefore and asyncAfter.
+- `camunda:asyncAfter`: Every task in Zeebe is always asyncBefore and asyncAfter.
+- `camunda:exclusive`: Jobs are always exclusive in Zeebe.
+- `camunda:jobPriority`: There is no way to prioritize jobs in Zeebe (yet).
+- `camunda:failedJobRetryTimeCycle`: You cannot yet configure the retry time cycle.
+
+### Service tasks
+
+![Service Task](../../components/modeler/bpmn/assets/bpmn-symbols/service-task.svg)
+
+Migrating a service task is described in detail in the section about adjusting your source code above.
+
+A service task might have **attached Java code**. In this case, the following attributes/elements are migrated and put into a task header:
+
+- `camunda:class`
+- `camunda:delegateExpression`
+- `camunda:expression` and `camunda:resultVariable`
+
+The topic `camunda-7-adapter` is set.
+
+- `camunda:failedJobRetryTimeCycle`: Here, the amount of defined retries is set to the `zeebe:taskDefinition retries` attribute.
+
```

Comment:
Only if you adapterEnabled is set to true.

Reply:
```suggestion
A service task might have **attached Java code**. In this case, the following attributes/elements are migrated and put into a task header:

- `camunda:class`
- `camunda:delegateExpression`
- `camunda:expression` and `camunda:resultVariable`

The topic `camunda-7-adapter` is set.

- `camunda:failedJobRetryTimeCycle`: Here, the amount of defined retries is set to the `zeebe:taskDefinition retries` attribute only if `adapterEnabled` is set to true.

```

@ManuelDittmar, did you mean for this statement (line 39) or the one in line 37?